styles.css

```diff
@@ -458,3 +460,399 @@ form {
 .message-label {
   padding-bottom: 40%;
 }
+
+.toolbar-links,
+.image-desktop,
+.about-project-desktop,
+.client-name-desktop,
+.name-desktop,
+.li-desktop,
+.project-desktop,
+.menu-logo {
+  display: none;
+}
+
+@media only screen and (max-width: 1440px) and (min-width: 768px) {
```

Comment:
- [ ] It appears that there may be an issue with the design, as the maximum width has been set to `1440px`. This is causing overflow for desktop screen sizes that are greater than 1440px. To address this issue, consider using the `min-width` property instead, with a value of `768px`, or changing the max-width property to min-width. Please ensure that the changes made work for larger screen sizes as well. I have reviewed the rest of the project at `1440px`, but it's important to ensure that the design is responsive and functional across greater screen sizes.
- Here's how it looks on my machine 👇
  ![image](https://github.com/agneta1991/Portfolio/assets/104968236/62ee80a9-e24d-4c36-9906-ae0747501a57)